### PR TITLE
Do get issue call right after modify issue call 

### PR DESCRIPTION
**Pull request by rmistry**
This is a workaround for the obfuscated emails issue

### CODE REVIEW

**Comment by rmistry** [comment]
PTAL @jonathanmetzman 
CC @micahbales

File: src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py [hunk 1]
```diff
@@ -73,8 +73,8 @@ def _sanitize_oses(oses):
   The OS custom field no longer has the 'Chrome' value.
   It was replaced by 'ChromeOS'.
   """
-  for i in range(len(oses)):
-    if oses[i] == 'Chrome':
+  for i, os in enumerate(oses):
```
Comment (jonathanmetzman) [inline src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py:76]: nit: call this os_field so it's not conlficting with Python's os module.

**Review by jonathanmetzman — APPROVED** [review]

File: src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py (hunk 1 above)
Comment (rmistry) [reply src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py]: Done.

**Pull request merged**